src/runtime/pkg/katautils/config-settings.go.in

```diff
@@ -117,3 +117,5 @@ const defaultPCIeSwitchPort = 0
 
 const defaultRemoteHypervisorSocket = "/run/peerpod/hypervisor.sock"
 const defaultRemoteHypervisorTimeout = 600
+
+const defaultSnpCertsPath = "/opt/snp/cert_chain.cert"
```

Comment:
Should we default to blank as this only applies to snp, not all hypervisors and the test of whether to append it is to check if it's not empty?

Reply:
The check is if it is not empty and it is an SNP guest.

Reply:
Ah, is the SNP guest check due to the `case snpProtection:` in `qemu_amd64.go`? That was collapsed, so I missed it